projects/iio_demo/Makefile.linux

```diff
@@ -0,0 +1,39 @@
+
+EXEC = iio_demo_linux
+PLATFORM = linux
+SYMBOLS = -DLINUX_PLATFORM
+BUILD_DIR = ./build_$(PLATFORM)
+PROJECT = $(realpath ./)
+NO-OS = $(realpath ../..)
+INCLUDE			= $(NO-OS)/include
+DRIVERS 		= $(NO-OS)/drivers
+PLATFORM_DRIVERS	= $(NO-OS)/drivers/platform/$(PLATFORM)
+
+CFLAGS += -DTINYIIOD_VERSION_MAJOR=0	 \
+		-DTINYIIOD_VERSION_MINOR=1		 \
+		-DTINYIIOD_VERSION_GIT=0x$(shell git -C $(NO-OS)/libraries/iio/libtinyiiod/ \
+						rev-parse --short HEAD) \
+		-DIIOD_BUFFER_SIZE=0x1000		 \
+		-D_USE_STD_INT_TYPES	\
+		-DTINYIIOD \
+		-DENABLE_IIO_NETWORK 
+CFLAGS += -DIIO_SUPPORT
+CFLAGS += -DDISABLE_SECURE_SOCKET
+
+include ./src.mk
+include  $(NO-OS)/tools/scripts/iio_srcs.mk
+
+all: copy $(EXEC)
+
+copy:
+	mkdir -p $(BUILD_DIR)
+	cp -r $(SRCS) $(BUILD_DIR)
+	cp -r $(INCS) $(BUILD_DIR)
+	cp -r $(SRC_DIRS) $(BUILD_DIR)
```

Comment:
I think at one point we changed from copy to symbolic link to avoid confusion when working with the files. Should this be similar?